homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,216 @@
+set_manual_level:
+  name: Set Manual Level
+  description: Sets manual level
+  fields:
+    manual_level:
+      name: Manual level
+      description: Manual level setting
+      required: true
+      advanced: false
+      default: "Off"
+      selector:
+        select:
+          options:
+            - "Off"
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+            - "Breeze"
+            - "Holiday"
+
+sync_time:
```

Comment:
Can be a `Button` entity?

Reply:
Removed sync_time here because implemented Button entity for this. #99494